n8n-db-provisioner.sh

```
@@ -0,0 +1,257 @@
+    log_info "User mapping saved to /tmp/user_mapping_$user_id.json"
+}
+
+# Step 4: Assign workflow to user
+assign_workflow_to_user() {
+    local workflow_id="$1"
+    local user_id="$2"
+    
+    log_info "Assigning workflow $workflow_id to user $user_id"
+    
+    # Get project ID for user
+    local project_id=$(exec_sqlite "SELECT id FROM project_entity WHERE name='clx_$user_id' LIMIT 1;")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)